### PR TITLE
New deployment

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,3 +11,11 @@ install:
 
 script:
   - npm run lint
+
+deploy:
+  provider: npm
+  email: "ben@daostack.io"
```
Comment (jellegerbrandy) [inline .travis.yml]: this will ony work for you :-)

File: .travis.yml [hunk 2]
```diff
@@ -11,3 +11,11 @@ install:
...
+  api_key: $NPM_API_TOKEN
```
Comment (jellegerbrandy) [inline .travis.yml]: need instructions on how to set this (`.env`? )

File: README.md [hunk 3]
```diff
@@ -251,5 +251,6 @@ In order to release a version:
    1. Make sure you have the required `.env` variables set (`kovan_provider`, `kovan_private_key`, `rinkeby_provider`,
       `rinkeby_private_key`).
    2. Make sure that you have a Dockerhub ID with permission to push an image to the `daostack` organization.
-   3. Make sure that you have an `npm` account with permission to push an image to the `daostack` organization.
-   4. `npm run release` - This will perform all the nesserary steps to deploy a new release.
+   3. Make sure that you are on your intended Git branch.
+   4. `npm run prepare-release` - This will perform all the nesserary steps to prepare a new release.
```
Comment (jellegerbrandy) [inline README.md]: We may say something about version numbers here (bc the script asks for it)

File: README.md [hunk 4]
```diff
@@ -251,5 +251,6 @@ In order to release a version:
...
+   5. After the branch is merged, Travis will automatically deploy a new version to NPM.
```
Comment (jellegerbrandy) [inline README.md]: merged where?

File: .travis.yml (hunk 1 above)
Comment (ben-kaufman) [reply .travis.yml]: I added it as Travis requires credentials of some account with publishing permissions, so I added mine.

File: .travis.yml (hunk 2 above)
Comment (ben-kaufman) [reply .travis.yml]: No need, this is already in travis and should only be executed by it

File: README.md (hunk 3 above)
Comment (ben-kaufman) [reply README.md]: Not anymore, the script will automatically increase the version now

File: README.md (hunk 4 above)
Comment (ben-kaufman) [reply README.md]: In the GitHub repo, the script will be executed when PR is merged into master

Comment (jellegerbrandy) [reply README.md]: ok, explain it in the docs, not to me :-)

**Comment by ben-kaufman** [comment]
@orenyodfat I don't think Travis support that type of complex conditioning...

**Pull request closed without merging**